### PR TITLE
ui: add checkout --relink to autocomplete scripts

### LINKED ISSUE #2983: checkout:  add `--relink` to tab completion scripts
> Per #2492 

~~And are there any other relatively newly added/changed/removed command options we should add to the tab completion files? Or commands, for that matter...~~ Moved to #2985

### PR DESCRIPTION
**Pull request by Suor**
Fix #2983.

### CODE REVIEW

File: scripts/completion/dvc.zsh [hunk 1]
```diff
@@ -72,8 +72,9 @@ _dvc_cache=(
 
 _dvc_checkout=(
   {-d,--with-deps}"[Checkout all dependencies of the specified target.]"
-  {-f,--force}"[Do not prompt when removing working directory files.]"
   {-R,--recursive}"[Checkout all subdirectories of the specified directory.]"
+  {-f,--force}"[Do not prompt when removing working directory files.]"
+  {--relink}"[Recreate links or copies from cache to workspace.]"
```
Comment (jorgeorpinel) [inline scripts/completion/dvc.zsh]: This is giving me
```console
$ dvc checkout - [TAB]
_arguments:comparguments:325: invalid argument: {--relink}[Recreate links or copies from cache to workspace.]
_arguments:comparguments:325: invalid argument: {--relink}[Recreate links or copies from cache to workspace.]
_arguments:comparguments:325: invalid argument: {--relink}[Recreate links or copies from cache to workspace.]
```

@Suor

Comment (jorgeorpinel) [reply scripts/completion/dvc.zsh]: I'm on ZShell BTW. Is it just me or is something missing from the script?

Comment (Suor) [reply scripts/completion/dvc.zsh]: Can you create a new issue for this @jorgeorpinel ?

Comment (jorgeorpinel) [reply scripts/completion/dvc.zsh]: Oh, well I already reopened the original one (see https://github.com/iterative/dvc/issues/2983#issuecomment-570462666) and then it was fixed by Ramon. All good, thanks!
p.s. please try to notice my mentions in the dvc.org repo, maybe you have notifications off? Thanks @Suor

**Pull request merged**